COMET.Web.Common/Utilities/HaveObjectChangedTracking/HaveObjectChangedTracking.cs

```diff
@@ -63,11 +63,36 @@ protected HaveObjectChangedTracking(ICDPMessageBus messageBus)
             this.MessageBus = messageBus;
         }
 
+        /// <summary>
+        /// Gets a <see cref="IReadOnlyCollection{T}" /> of added <see cref="Thing" />s
+        /// </summary>
+        public IReadOnlyCollection<Thing> GetAddedThings => this.AddedThings;
```

Comment:
Hmm surprised to see the word Get in a property.... I would suggest something else like *Value -> AddedThingsValue